lib/metapackage.py

```diff
@@ -5,9 +5,13 @@
 from lib import exception
 from lib import packages_manager
 from lib import rpm_package
+from lib import utils
 from lib.versions_repository import replace_file_section
 
 LOG = logging.getLogger(__name__)
+DIST_MACRO_NAME = "%{dist}"
+DIST_MACRO_VALUE = utils.run_command(
+    "rpm --eval {}".format(DIST_MACRO_NAME)).strip()
```

Comment:
This will evaluate the `%dist` macro on the host system, not inside mock environment, where packages are actually built.

Don't you think we should eval `%dist` inside mock environment?